terraform/runbook.tf

```diff
@@ -45,5 +57,24 @@ resource "azurerm_automation_module" "az_keyvault_module" {
     uri = "https://devopsgallerystorage.blob.core.windows.net/packages/az.keyvault.4.0.2-preview.nupkg"
   }
   # This module cannot be imported unless the Accounts Module has been imported first
-  depends_on = [ azurerm_automation_module.az_accounts_module ]
+  depends_on = [azurerm_automation_module.az_accounts_module]
+}
+
+resource "azurerm_automation_certificate" "github_membership_runas_cert" {
+  name                    = "AzureRunAsCertificate"
+  resource_group_name     = azurerm_automation_account.github_membership_automation.resource_group_name
+  automation_account_name = azurerm_automation_account.github_membership_automation.name
+  base64                  = data.azurerm_key_vault_secret.runas_cert_secret.value
```

Comment:
Had to get the cert using a azurerm_key_vault_secret  data source to get the full pfx version